_opensearch/remote.md

```diff
@@ -0,0 +1,120 @@
+{: .note}
+
+## Enable the feature
+
+In order to create new indexes with remote-backed storage enabled, you must first enable these features by adding the correct properties to `run.gradle` before building OpenSearch. See the [developer guide](https://github.com/opensearch-project/OpenSearch/blob/main/DEVELOPER_GUIDE.md) for information about how Gradle is used to build OpenSearch.
+
+Add the following properties to `run.gradle` to enable the feature:
+
+```bash
+testClusters {
+  runTask {
+    testDistribution = 'archive'
+    if (numZones > 1) numberOfZones = numZones
+    if (numNodes > 1) numberOfNodes = numNodes
+    systemProperty 'opensearch.experimental.feature.replication_type.enabled', 'true'
+    systemProperty 'opensearch.experimental.feature.remote_store.enabled', 'true'
+  }
+}
+```
+
+Segment replication must also be enabled in order to use remote-backed storage.
+{: .note}
+
+After building OpenSearch with these properties, you must enable the feature for all nodes in your cluster. This can be done by modifying a `config/jvm.options`, or by defining `OPENSEARCH_JAVA_OPS` from the command line.
+
+### Option 1: Modify jvm.options
+
+Add the following lines to `config/jvm.options` before starting the OpenSearch process to enable the feature and its dependency:
+
+```
+-Dopensearch.experimental.feature.replication_type.enabled=true
+-Dopensearch.experimental.feature.remote_store.enabled=true
+```
+
+Run OpenSearch
+
+```bash
+./bin/opensearch
+```
+
+### Option 2: Enable from an environment variable
+
+As an alternative to directly modifying `config/jvm.options`, you can define the properties by using an environment variable. This can be done in a single command when you start OpenSearch or by defining the variable with `export`.
+
+To add these flags in-line when starting OpenSearch:
+
+```bash
+OPENSEARCH_JAVA_OPTS="-Dopensearch.experimental.feature.replication_type.enabled=true -Dopensearch.experimental.feature.remote_store.enabled=true" ./opensearch-{{site.opensearch_version}}/bin/opensearch
+```
+
+If you want to define the environment variable separately, prior to running OpenSearch:
+
+```bash
+export OPENSEARCH_JAVA_OPTS="-Dopensearch.experimental.feature.replication_type.enabled=true -Dopensearch.experimental.feature.remote_store.enabled=true"
+./bin/opensearch
+```
+
+## Register a remote repository
+
+Now that your deployment is running with the feature flags enabled, the next step is to register a remote repository where backups will be stored. See [Register repository]({{site.url}}{{site.baseurl}}/opensearch/snapshots/snapshot-restore#register-repository) for more information.
+
+## Create an index
+
+Remote-backed storage is enabled for an index when it is created. This feature cannot be enabled for indexes that already exist.
```

Comment:
Is it possible to enable it if you use reindexing for an existing index? Like segrep?

Reply:
No idea. This was not part of the feature brief so I assume not.